src/main/java/io/appium/java_client/ws/MessagesHandler.java

```diff
@@ -0,0 +1,28 @@
+package io.appium.java_client.ws;
+
+/**
+ * This is the basic interface for all web socket message handlers.
+ */
+public interface MessagesHandler {
```

Comment:
It's strange that messages handler can not handle messages.

Reply:
yep, I'll make it generic